source/features/more-dropdown.tsx

```diff
@@ -55,55 +38,34 @@ async function init(): Promise<void> {
 	const dependenciesUrl = `/${repoUrl}/network/dependencies`;
 
 	const nav = select('.js-responsive-underlinenav .UnderlineNav-body');
-	if (nav) {
-		// "Repository refresh" layout
-		nav.parentElement!.classList.add('rgh-has-more-dropdown');
-		select('.js-responsive-underlinenav-overflow ul')!.append(
-			<li className="dropdown-divider" role="separator"/>,
-			createDropdownItem('Compare', compareUrl),
-			pageDetect.isEnterprise() ? '' : createDropdownItem('Dependencies', dependenciesUrl),
-			createDropdownItem('Commits', commitsUrl),
-			createDropdownItem('Branches', `/${repoUrl}/branches`)
-		);
+
```

Comment:
No need to remove the previous version yet, I think. Code is split cleanly so they can co-exist.

Reply:
My guess is that you wanted to use `createDropdownItem`, but I think that’s wrong. The current version does not use `li` elements, which is why we just used JSX inline instead of `createDropdownItem`

Reply:
> The current version does not use `li` elements

Repository tabs
![Screenshot_20200819_093017](https://user-images.githubusercontent.com/36174850/90600171-c322c500-e1fe-11ea-9dd0-7b11d048fe16.png)
Dropdown
![Screenshot_20200819_093052](https://user-images.githubusercontent.com/36174850/90600176-c4ec8880-e1fe-11ea-948d-0016e6a63c32.png)

Reply:
My bad, I had misread the code